app/models/agents/webhook_agent.rb

```diff
@@ -20,6 +20,9 @@ class WebhookAgent < Agent
         * `payload_path` - JSONPath of the attribute in the POST body to be
           used as the Event payload.  If `payload_path` points to an array,
           Events will be created for each element.
+        * `verbs` - Comma-separated list of http verbs your agent will accept.
```

Comment:
That's fine, I'll go with commas.  It's what I documented.  I'll make the change soon.

I've had this come up **once** with a client. Instructions clearly stated comma, they went semicolon.  They were wrong, software was right... or was it?  Their intent was pretty clear.  They switched to commas, but the next release had hidden support for semi-colons.  

It was a good lesson.  Maybe whatever decision two programmers arbitrarily make in a pull request on September 1st shouldn't be the gospel?